test/geometry/GeoTrackView.test.cc

```diff
@@ -71,55 +71,55 @@ TEST_F(GeoTrackViewHostTest, track_line)
 
     {
         // Track from outside detector, moving right
-        geo = {{-10, -10, -10}, {1, 0, 0}};
-        EXPECT_EQ(VolumeId{0}, geo.volume_id()); // Shape2 center
+        geo = {{-100, -100, -100}, {1, 0, 0}};
+        EXPECT_EQ(VolumeId{3}, geo.volume_id()); // Shape2 center
```

Comment:
The change in your GeoParams tests shows the volume IDs are "implementation dependent". Instead of changing them like this (which will cause test failures  if the ROOT implementation is changed), use:
```suggestion
        EXPECT_EQ(this->params()->label_to_id("Shape2"), geo.volume_id());
```

Also why did the positions change? Is this  a cm/mm thing? That *absolutely* needs to be encapsulated by the geo params implementation.

Reply:
I observed that the assigment of volume IDs changes between vgdml and Root.
Encapsulating this change into `GeoParams` needs some thinking.